### DISCUSSION TITLE
Gnome 41 Power Profiles

### DISCUSSION THREAD

**Discussion by ejawhary** [AdnanHodzic/auto-cpufreq#283] [Q&A]
I just got Gnome 41 with a full package upgrade as I am on Arch. As you probably already know it has a new power profiles feature. My question is will Auto CPU Freq and Gnome Power Profiles work against each other? I wish to have my laptop on full performance mode when it is plugged in and battery saver when it is on battery. Thanks

**Comment by AdnanHodzic** [comment]
This was mentioned in #277, for me on Ubuntu Gnome 40 I don't have any problems. But it seems like this is an issue for others on Gnome 41. As mentioned in that issue, there's a workaround and I also proposed how to permanently fix it.
